packages/bus-sqs/src/sqs-transport.ts

```diff
@@ -563,6 +596,71 @@ export class SqsTransport implements Transport<SQSMessage> {
       )
     return delay / MILLISECONDS_IN_SECONDS
   }
+
+  private async assertSnsSqsSubscriptionByArn(
+    topicArn: string,
+    sqsQueueArn: string
+  ) {
```

Comment:
nit - can this define it's return type to help readability?

either 

```suggestion
  ): Promise<true> {
```

that matches the current form, or

```suggestion
  ): Promise<void> {
```

since the return type isn't necessary if this is an assertion that throws an error when invalid.

ditto for the remaining methods